components/ActivityIndicator/README.md

```diff
@@ -86,88 +69,157 @@ import MaterialComponents
 ```
 <!--</div>-->
 
-### Indeterminate indicators
-Indeterminate indicators visualize an unspecified wait time.
-
-### Determinate indicators
-Determinate indicators display how long an operation will take.
-
 - - -
 
+## Overview
+
+`MDCActivityIndicator` is a view that has two modes: indeterminate and determinate. Indeterminate
+indicators express an unspecified wait time, while determinate indicators represent the length of a
+process. Activity indicators are indeterminate by default.
 
-## Examples
+## Usage
 
+### Typical use: Indeterminate
 
-### Indeterminate indicators
-When indicators are indeterminate they request that the user wait while something finishes when it's not necessary to indicate how long it will take. This is the default mode and no additional parameters need to be set.
+MDCActivityIndicator instances are indeterminate by default.
 
 <!--<div class="material-code-render" markdown="1">-->
 #### Swift
 ``` swift
-let activityIndicator = MDCActivityIndicator(frame: CGRect(x: 0, y: 0, width: 32, height: 32))
+let activityIndicator = MDCActivityIndicator()
+activityIndicator.sizeToFit()
 view.addSubview(activityIndicator)
 
 // Start animation
 activityIndicator.startAnimating()
-...
+
```

Comment:
Why should they be calling `stopAnimating()` immediately after calling `startAnimating()`?

Reply:
Done.